channel_vis.py

```diff
@@ -0,0 +1,187 @@
+import torch
+from torchvision import datasets, models
+import numpy as np
+import cv2
+import matplotlib.pyplot as plt
+import argparse
+import sys
+import os
+
+from utils.activation_analysis import LayerActivationAnalysis
+from models.UNet import UNet
+
+# Maps an encoder layer index to the index of the child module associated with
+# that layer in the VGG11 model
+vgg11_layer_dict = {1:0, 2:3, 3:6, 4:8, 5:11, 6:13, 7:16, 8:18}
+
+def get_conv_layer(model, layer_index):
```

Comment:
It's often nice to comment what a function does, though in some cases it's not necessary (if it's blindingly obvious). Something like:
```
def get_conv_layer(model, layer_index):
    """Get the layer specified by layer_index from the model"""
    # Code here
```

Reply:
Good suggestion, Connor.  Just added the comments.